tutorials/solution_strategies.ipynb

```diff
@@ -0,0 +1,224 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #88.        pp.models.solution_strategy.ContactIndicators,
</pre><p>Here I suggest a short remark that this is the mixin that makes the model and the line search talk, while the rest is standard.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/pmgbergen/porepy/pull/1208/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>

Reply:
Done.